### DISCUSSION TITLE
Multi select limit by one per group

### DISCUSSION THREAD

**Discussion by horacioh** [ariakit/ariakit#3072] [Questions] [answered]
hey folks!

have someone tried to build a multiselect that limits to one selection per group or items like this color picker in Notion?
<img width="936" alt="Screenshot 2023-11-21 at 00 07 35" src="https://github.com/ariakit/ariakit/assets/725120/f4212538-45bc-4c54-b1bb-c236691ca810">

thanks in advanced!

**Comment by diegohaz** [comment, marked as answer]
There's no native concept of a multiselect element that only allows one selection per group.

In this situation, I'd use a [Menu](https://ariakit.org/components/menu) with [MenuItemRadio](https://ariakit.org/examples/menu-item-radio) elements grouped into [`MenuGroup`](https://ariakit.org/reference/menu-group)s.
